package.json

```diff
@@ -39,7 +39,6 @@
     "concat-stream": "^1.5.1",
     "coveralls-send": "0.0.2",
     "cross-spawn-async": "^2.1.9",
-    "documentjs": "^0.4.1",
```

Comment:
Unless there's something I'm missing, this didn't do anything. `documentjs` is still added as a devDependency of projects created with `donejs add app` by this line: https://github.com/donejs/cli/blob/major/package.json#L82.